Cython/Compiler/ParseTreeTransforms.py

```diff
@@ -846,6 +850,16 @@ def visit_NameNode(self, node):
             directive = self.directive_names.get(node.name)
             if directive is not None:
                 node.cython_attribute = directive
+        if node.as_cython_attribute() == "compiled":
```

Comment:
A test for the plain `NameNode` case seems to be missing. All tests use `cython.compiled`, none uses something like `from cython import compiled` (and I'm not even sure that that is tested anywhere else yet…).